apps/site/pages/components/molecules/alert.mdx

```diff
@@ -112,7 +140,7 @@ import '@faststore/ui/src/components/atoms/Alert/styles.scss'
 
 ## Props
 
-<PropsSection name="BuyButton" />
+<AlertPropsSection component="Alert" />
```

Comment:
It would be interesting to add link to `LinkProps` (But we still don't have this component props yet hahaha) 

<img width="884" alt="image" src="https://github.com/vtex/faststore/assets/3356699/c1714bd0-bec4-4f0c-a6e6-f5a531539861">